community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/BatchingTransactionAppender.java

```diff
@@ -93,11 +93,6 @@ public void start() throws Throwable
     @Override
     public long append( TransactionToApply batch, LogAppendEvent logAppendEvent ) throws IOException
     {
-        // We put log rotation check outside the private append method since it must happen before
```

Comment:
It seems that it wasn't true, right? And the comment says why? Also it isn't like that anymore, now with the new changes.